pkg/ccn-lite/Makefile

```diff
@@ -11,14 +11,14 @@ TOOLCHAIN_FILE=$(PKG_BUILDDIR)/xcompile-toolchain.cmake
 
 all: $(PKG_BUILDDIR)/src/Makefile
 	$(MAKE) -C $(PKG_BUILDDIR)/src && \
-	cp $(PKG_BUILDDIR)/src/lib/libccnl-riot.a ${BINDIR}/ccn-lite.a
+	cp $(PKG_BUILDDIR)/src/lib/libccnl-riot.a $(BINDIR)/ccn-lite.a
```

Comment:
unrelated

Reply:
Yeah I applied https://github.com/RIOT-OS/RIOT/pull/8822 here too, I will move to another PR.

Reply:
Moved to https://github.com/RIOT-OS/RIOT/pull/8907